Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -31,10 +31,53 @@
 // Significantly modified for re-motion
 // ************************************************
 
+interface JQueryStatic { Autocompleter: any; }
```

Comment:
@Stefan-Ilic , updated with @a-ctor: We're going to keep the jquery.js file in this branch and move the typescript version to a new PR @a-ctor will use to do his part-2 of the file. the intention has always been to invest the minimal effort needed to get typescript to work (just sprinkle some annotations on it). Since we don't merge all files into a single typescript file right away, we can defer changing the jquery.js file to typescript. This allows us to do a single, cohesive update of the file instead of dealing with all those half-truths in the typesystem.

Reply:
Alright, then I will revert the file and disregard all comments concerning it

Reply:
> 
> 
> Alright, then I will revert the file and disregard all comments concerning it

Please don't forget to create a backup of the file in a separate branch. @a-ctor will probably want to start his work off yours :)

Reply:
> 
> 
> I also thought the intention is to do the minimum work to make TypeScript work and simply sprinkle annotations on it, which is why many of your comments regarding this file confused me, since they seemed to contradict that notion, at least to me

Yeah, there was just way more work done and way less typing for the work involved to make sense any more. But mostly, with the option to just not create the bundled file right now, a lot of the constraints went out the window.